docs/source/schema/creating-directives.mdx

```diff
@@ -26,8 +26,7 @@ type Query {
 >   <img alt="Edit upper-case-directive" src="https://codesandbox.io/static/img/play-codesandbox.svg" />
```

Comment:
The sandbox still uses the old API. Should you update it too?

Reply:
Yep! Oops forgot to create the PR until just now, but was going to merge these two simultaneously: https://github.com/apollographql/docs-examples/pull/32